kitura-init.js

```diff
@@ -29,158 +26,27 @@ if (projName.charAt(0) === '.') {
-    }
-    // Rename instances of project name (charts can't contain special characters)
-    var renameClean = spawnSync('find', ['.', '-exec', 'sed', '-i', '', 's/' + oldProjNameClean + '/' + projNameClean + '/g', '{}', ';'])
-    if (renameClean.status !== 0) {
-        console.error(chalk.red('Error: ') + 'could not rename project.');
-        console.error(renameClean.stderr.toString());
-        process.exit(renameClean.status);
-    }
-    var renameLowercase = spawnSync('find', ['.', '-exec', 'sed', '-i', '', 's/' + oldProjNameLowercase + '/' + projNameLowercase + '/g', '{}', ';'])
-    if (renameClean.status !== 0) {
-        console.error(chalk.red('Error: ') + 'could not rename project.');
-        console.error(renameClean.stderr.toString());
-        process.exit(renameClean.status);
-    }
-    var renameLowercaseClean = spawnSync('find', ['.', '-exec', 'sed', '-i', '', 's/' + oldProjNameCleanLowercase + '/' + projNameCleanLowercase + '/g', '{}', ';'])
-    if (renameClean.status !== 0) {
-        console.error(chalk.red('Error: ') + 'could not rename project.');
-        console.error(renameClean.stderr.toString());
-        process.exit(renameClean.status);
-    }
-    // Rename instances of project name which can't contain Uppercase characters
-    var rename = spawnSync('find', ['.', '-exec', 'sed', '-i', '', 's/' + oldProjName + '/' + projName + '/g', '{}', ';'])
-    if (rename.status !== 0) {
-        console.error(chalk.red('Error: ') + 'could not rename project.');
-        console.error(rename.stderr.toString());
-        process.exit(rename.status);
-    }
-}
-
-function buildProject() {
-    console.log('Running `swift build` to build project...');
-
-    var build = spawnSync('swift', ['build'], { stdio:[0,1,2] });
-    if (build.status !== 0) {
-        console.error(chalk.red('Failed to complete build.'));
-        process.exit(build.status);
-    } else {
-        console.log('Running `swift package generate-xcodeproj` to generate Xcode Project...');
-
-        var xcodeProj = spawnSync('swift', ['package', 'generate-xcodeproj'], { stdio:[0,1,2] });
-        if (xcodeProj.status !== 0) {
-            console.error(chalk.red('Failed to generate Xcode Project.'));
-            process.exit(xcodeProj.status);
-        } else {
-            console.log(chalk.green('Project built successfully.'));
-            console.log('Next steps:');
-            console.log('');
-            console.log('Open your Xcode project:');
-            console.log(chalk.grey('   $ open ' + projName + '.xcodeproj'));
-            console.log('');
-            console.log('Or, run your app from the terminal:');
-            console.log(chalk.grey('   $ swift run'));
-        }
-    }
-}
-
-
-
 function printHelp() {
     console.log("");
     console.log("  Usage: kitura init [options]");
     console.log("");
-    console.log("  Scaffold a bare-bones Kitura project.");
+    console.log("  Scaffold a basic cloud ready Kitura project.");
```

Comment:
Delete "basic" because we also have `kitura basic` and we don't want to cause any (more) confusion.